joomla/joomla-cms
# [4.0] Cassiopeia - Add overflow hidden to ul and ol

Pull Request for Issue #33501

### Summary of Changes
Added `overflow: hidden` to `<ul>` and `ol` globally.
Lists that are to the right of images having `float: left` retain their indentations for li, all other lists are not affected by this.


### Testing Instructions
Prerequisites: Install blog sample data
- Apply Patch
- `npm run build:css`
- Frontend: Visit the blog 'Your Modules' and check the list indentation
- Also, check any blog article that has a list without an image. In this case, everything should remain as is.


### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/53610833/117174331-3b600380-adeb-11eb-8a17-02ad55c2dad5.png)


### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/53610833/117174171-11a6dc80-adeb-11eb-95e2-7446954bff93.png)


### Documentation Changes Required
None
